src/main/kotlin/no/skatteetaten/aurora/cantus/controller/commands.kt

```diff
@@ -57,4 +62,15 @@ class ImageRepoDtoAssembler(
             throw BadRequestException("Invalid Docker Registry URL url=$urlToValidate")
         }
     }
-}
\ No newline at end of file
+
+    private fun String.splitCheckNull(): List<String?> {
```

Comment:
Ville det vært mulig å endre denne til `List<String>?`
Ser litt rart ut å populere en liste med bare null verdier.

Reply:
Ikke så lenge det blir brukt destructuring. Da må jeg ha en ekstra try catch isåfall

Reply:
ah, godt poeng. da må du ha en liste med 4 elementer da